testing/thunderbird-extension-enigmail/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=thunderbird-extension-enigmail
+pkgver=2.0.12
+pkgrel=1
+pkgdesc="OpenPGP message encryption and authentication for Thunderbird"
+url="https://www.enigmail.net/"
+arch="noarch"
+license="MPL GPL3"
+depends="thunderbird gnupg"
+makedepends="zip python2 perl"
+options="!check"
+source="https://www.enigmail.net/download/source/enigmail-${pkgver}.tar.gz
+        0001-preferences-disable-pEpAutoDownload-by-default.patch"
+builddir="${srcdir}/enigmail"
+
+prepare() {
```

Comment:
```suggestion
```

Reply:
Thanks for the review!

Reply:
You're welcome! Thank you for contributing!